api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/TaskTranslatorImplTest.java

```diff
@@ -22,10 +23,16 @@
 @RunWith(MockitoJUnitRunner.class)
```

Comment:
Not that it's harmful, but we don't need this if we aren't using the `@Mock` annotation